.circleci/config.yml

```diff
@@ -148,6 +175,39 @@ jobs:
           tag: $CIRCLE_TAG
           google-project-id: GOOGLE_PROJECT_ID
 
+      # this could be done by re-tagging the existing image but for ease of migration
+      # I'm duplicating it fully with the intent of removing the above steps entirely
+      # /moz-fx-glam-prod/glam-prod/glam@sha256:705f53c5511c5803c6ea58896a93a97a30dc8defdef19c5d2243c3171a13be25
+      - run:
```

Comment:
Do you need to do this if you're just using the `gcp-gcr`  orb? You're not actually using the GCP CLI after you set it up.  It seems like you should be able to get away with a simpler config than this that's just `gcp-gcr/build-and-push-image` with the right environment variables set and with `use_oidc` set to `true`. But if this works then that's fine.